bref

```diff
@@ -31,28 +31,19 @@ $app->command('init', function (SymfonyStyle $io) {
         return 1;
     }
 
-    $configured = true;
     if (getenv('AWS_ACCESS_KEY_ID') === false || getenv('AWS_SECRET_ACCESS_KEY') === false) {
         // Credentials are not configured with environment variables, check aws cli
-        if ($exeFinder->find('aws') !== null) {
```

Comment:
This is not needed, `Process` will exits with status 1 if not found. It simplifies the block :)